ui/pages/settings/advanced-tab/advanced-tab.component.js

```diff
@@ -566,10 +566,7 @@ export default class AdvancedTab extends PureComponent {
   }
 
   handleIpfsGatewaySave() {
```

Comment:
Why this change?

Reply:
As discussed on a call, we need a migration to properly support this and an update to the `attemptResolve` function in the background

The reason this change is needed is because the controller now expects the ipfs gateway to be a url

Reply:
should be addressed in https://github.com/MetaMask/metamask-extension/pull/12873/commits/a3732fcc6bc02e22c12fcd8450660f12963c4ad7